src/core/tetrisBoard.ts

```diff
@@ -226,84 +239,83 @@ export class TetrisBoard {
    * @return: Optimal Y for the ghost tetromino
    */
   public findGhostTetrominoY(corX: number, corY: number,
-    type: TetrominoManager.TetrominoType,
-    rotation: TetrominoManager.TetrominoRotation,
+    type: TetrominoType, rotation: TetrominoRotation,
   ): number {
-    /* First we find the lowest Y among the number of cols this tetromino
-    spans */
-    let yHigherThanCmp = false;
-    const yToCmpArr: number[] = [];
-
-    const tetrominoCoords =
-      TetrominoManager.TetrominoManager.getTetrominoCoords(type, rotation);
-    for (let pixelIter = 0; pixelIter < MAX_PIXEL; pixelIter += 1) {
-      const coord = tetrominoCoords[pixelIter];
-      const xToCheck = corX + coord[TetrominoManager.X_INDEX];
-      const yToCheck = corY + coord[TetrominoManager.Y_INDEX];
+    let retGhostCorY = 0;
+    if (type < TetrominoType.Ghost) {
```

Comment:
This might look like a huge change but I just added some input sanitization here (wrapped everything in a `if (type < TetrominoType.Ghost)` check